gef.py

```diff
@@ -3639,21 +3635,26 @@ def parse_string_range(s):
 
 @lru_cache()
 def gef_get_auxiliary_values():
-    """Retrieves the auxiliary values of the current execution. Returns None if not running, or a dict()
-    of values."""
+    """Retrieves the ELF auxiliary values of the current execution. This information is provided by
```

Comment:
Why is there 2 docstrings ?

Reply:
oh, don't know how that happened. will fix

Reply:
The one starting at L3638 then at L3640